tables/de-g0-bd.utb

```diff
@@ -0,0 +1,91 @@
+# liblouis: German Grade 0 Braille (bidirectional
+#
+# Das System der deutschen Blindenschrift (2015/2018)
+# http://www.bskdl.org/textschrift.html
+#
+# Bemerkung:
+# - Gross-/Kleinschreibung wird nicht berücksichtigt (d.h. für
+#   Abkürzungen u.ä. braucht es Vorprogramm)
+# - Akzente: Hinweis auf die beiden Tabellen (bis jetzt nur als
+#   Kommentar in de-g0.utb)
+#
+# -----------
+# Differences between this table (de-g0-bd.utb) and de-g0.utb
+# This table is meant to be used with screen readers and note-takers
+# for both forward and backward translation.
+# Forward translation in this table is designed to give as much
+# information as possible in order for the result to be
+# back-translatable.
+# Forward translation differs from that of de-g0.utb in the following ways.
+# 1. All capital letters are marked.
+# 2. Accented letters are translated using de-accents-detailed.cti
+#    to make the translation as detailed as possible.
+# 3. "`" and "´" are translated as "'" so they won't cover other
+#    characters.
+# -----------
+
+#-name: Deutsche Basisschrift
+#-display-name: German uncontracted braille
+#
+#+locale:de
+#+type:literary
+#+contraction:no
+#+grade:0
+#+direction:both
+# ------------
+#
+#  Copyright (C) 2018 SBS Schweizerische Bibliothek für Blinde, Seh- und Lesebehinderte
+#  Copyright (C) 2020 Bue Vester-Andersen
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+#
+#-------------------------------------------------------------------------------
+
+# must include display table directly to establish Character/dots associations.
+include de-eurobrl6.dis
```

Comment:
This is a bit fishy.

Reply:
> Then, what do you want? Some display statements need to be there to define the dot patterns. The characters themselves have other definitions in a literary (none-computer Braille) table.

I think I would prefer if the display rules that are actually needed are inserted here instead of the include. But more importantly I'd like to understand the issue better. Which are the dot patterns that need to be defined and why?

Reply:
Changed the comment